src/elements/content-explorer/ItemGridCell.js

```diff
@@ -24,10 +26,20 @@ const ItemGridCell = ({
     view,
     ...rest
 }: Props) => {
+    const url = getProp(item, `thumbnailUrl`);
```

Comment:
Is the `getProp` usage here meant to address a concern that `item` is undefined? Does that affect anything else?

Reply:
`item` should always be defined. The `getProp` here is because `thumbnailUrl` may not be defined.  I check later on whether this url value is defined or not.